photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -75,10 +69,76 @@ final class Driver(val sc: SparkContext, val params: GameParams, implicit val lo
 
     val normalizationContexts = Timed("Prepare normalization contexts") {
       prepareNormalizationContexts(trainingData, featureIndexMapLoaders, featureShardStats)
+        .map(_.mapValues(context => sc.broadcast(context)))
```

Comment:
Move later so that we don't have to broadcast if e.g. preparing the training data set fails?